### PR TITLE
README.md updates, main-old branch

### PR DESCRIPTION
**Pull request by johnkerl**

### CODE REVIEW

File: README-docs.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Temporary and experimental.
 
 # Rationale
 
-* R docs (currently at [https://github.com/TileDB-Inc/tiledbsoma](https://github.com/TileDB-Inc/tiledbsoma)) have a wonderful combination of API docs (generated from in-source-code doc-blocks) as well as hand-written long-form "vignette" material.
+* R docs (currently at [https://github.com/TileDB-Inc/TileDB-SOMA](https://github.com/TileDB-Inc/TileDB-SOMA)) have a wonderful combination of API docs (generated from in-source-code doc-blocks) as well as hand-written long-form "vignette" material.
```
Comment (bkmartinjr) [inline README-docs.md:7]: wrong repo name (see the organization part of the path)

File: apis/python/examples/examples-overview.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ In these example we will offer how-to's on the [TileDB SingleCell Python package
 
 See also for the R package:
 
-* [R repo](https://github.com/TileDB-Inc/tiledbsoma)
+* [R repo](https://github.com/TileDB-Inc/TileDB-SOMA)
```
Comment (bkmartinjr) [inline apis/python/examples/examples-overview.md:31]: repo name error

File: README.md [hunk 3]
```diff
@@ -6,6 +6,23 @@
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main-old`, implements the [original specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md). Please also see the `main` branch which implements the [updated specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).
+
+# Language bindings and installation instructions
+
+* [Python](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/python)
+* [R](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/r)
+* [API documentation](https://tiledb-inc-tiledb-soma.readthedocs-hosted.com/en/latest/)
```
Comment (bkmartinjr) [inline README.md:15]: does not need to be resolved to land this PR, but should we consider a separate RTD domain/account for SOMA?   CC:  @ambrosejcarr @aaronwolen

File: apis/python/examples/ingesting-data-files.md [hunk 4]
```diff
@@ -105,7 +105,7 @@ queries benefit from that optimized storage. In particular, various cross-corpus
 in these examples take just seconds or minutes.
 
 A key point is **write once, read from multiple tools** -- in particular, using `tiledbsoma-py` (this
-package) or [`tiledbsoma-r`](https://github.com/TileDB-Inc/tiledbsoma) you can read SOMAs in either
+package) or [`tiledbsoma-r`](https://github.com/TileDB-Inc/TileDB-SOMA) you can read SOMAs in either
```
Comment (bkmartinjr) [inline apis/python/examples/ingesting-data-files.md:108]: is the repo path wrong?

File: README.md [hunk 5]
```diff
@@ -6,6 +6,23 @@
...
+
+# Issues and contacts
+
+* Any/all questions, comments, and concerns are welcome at https://github.com/single-cell-data/TileDB-SOMA/issues
+* Bug reports should ideally be accompanied by a minimally reproducible example, using sample data available outside your organization
+
```
Comment (bkmartinjr) [inline README.md:21]: suggest similar security notification addition that you made to the other README

**Review by bkmartinjr — DISMISSED** [review]
couple of minor issues: 
* Repo org wrong in a couple of spots
* maybe add a security notification POC?

File: README.md (hunk 3 above)
Comment (johnkerl) [reply README.md:15]: @bkmartinjr we are on that (internal discussions well in place)

File: apis/python/examples/ingesting-data-files.md (hunk 4 above)
Comment (johnkerl) [reply apis/python/examples/ingesting-data-files.md:108]: @bkmartinjr i've found & fixed several issues on more recent commits -- thank you!

File: README.md (hunk 5 above)
Comment (johnkerl) [reply README.md:21]: yup, on the way! :)

**Comment by johnkerl** [comment]
The `R-CMD-check` fail is unrelated to this PR -- I will investigate separately

**Review by bkmartinjr — APPROVED** [review]

**Comment by johnkerl** [comment]
I'll merge this as-is (as a `README.md` update) but will continue pursuing the R CI fail. We'll need that fixed before cutting a new release on `main-old` -- our first since the repo merge of #285 (see also #289 and #292).

Very sorry @aaronwolen I wanted this in ship-shape when you returned from PTO; alas it is not but I'll keep digging intoit.

**Pull request merged**